autoref/rhapi.py

```diff
@@ -103,30 +104,20 @@ def get(self, parts, data = None, headers = None, params = None, verbose = False
 
         self.dprint(callurl, "with payload", sdata, "and headers", headers)
 
-        req = urllib2.Request(url = callurl, data = data)
-        if headers != None:
-            for h in headers.keys():
-                req.add_header(h, headers[h])
+        resp = None
+        if(data == None):
+            resp = requests.get(callurl, headers=headers)
+        else:
+            resp = requests.post(callurl, headers=headers, data=data)
 
-        resp = urllib2.urlopen(req)
+        if self.debug:
+                self.dprint("Response", resp.status, " ".join(resp.text).split("\r\n"))
```

Comment:
Is this an indent error? Looks like 8 spaces rather than 4.

Reply:
True, nicely spotted! That's indeed an over indentation. It will not throw and error when running, but it is better to fix it now.